linux.cc

```diff
@@ -396,6 +397,22 @@ static long sys_getcpu(unsigned int *cpu, unsigned int *node, void *tcache)
     return 0;
 }
 
+#define __NR_sys_prlimit __NR_prlimit64
+static int sys_prlimit(pid_t pid, int resource, const struct rlimit *new_limit,
+    struct rlimit *old_limit)
+{
+    if (new_limit) {
+        return 0;
```

Comment:
Why return 0? If the new limits are ignored maybe it should return EINVAL or EPERM or something? And at least warn_once that it's unimplemented?

Reply:
Will fix it in a new PR.